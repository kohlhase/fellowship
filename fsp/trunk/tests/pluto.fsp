lj. minimal.

(* fully bottom-up *)
declare A,B,C,D : bool.
goal (A -> (A -> B) -> (A -> C) -> (B -> C -> D) -> D).
elim a.
elim ab.
elim ac.
elim bcd.
cut (A) d.
axiom.
cut (A -> B) a2.
axiom.
elim.
axiom.
cut (A -> C) b.
axiom.
elim.
axiom.
cut (B -> C -> D) c.
axiom.
elim.
axiom.
elim.
axiom.
axiom.
export natural language.
checkout proof term isabelle.
discard.
