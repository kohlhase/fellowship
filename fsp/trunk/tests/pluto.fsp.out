
    > Current logic: intuitionistic sequent calculus.
      

    > Current logic: intuitionistic sequent calculus.
      

    > Current logic: minimal intuitionistic sequent calculus.
      
fsp < fsp < fsp < 
    > A,B,C,D defined.
      
fsp < Line 5, character 1:
.Parse error: syntactical analysis.
